modules/deseq2.nf

```diff
@@ -3,6 +3,7 @@
 ***************************************************/
 process deseq2 {
     label 'deseq2'
+    label 'smallTask'
```

Comment:
deseq2 can become quite compute extensive for large data sets and many comparisons I think... but let's try it

Reply:
what would you suggest for `cpus` and `memory`?

Reply:
I think `memory` can become an issue for a large read count matrix and many comparisons. But we need to test this. And it does not run multi-threaded anyway if I am right. so maybe `cpu=1` and `memory=4.G` as a start configuration? and then we see if it crashes (the SPP studies are good test cases)